Gemfile

```diff
@@ -60,6 +60,7 @@ gem "secure_headers",                 "~>3.0.0"
 gem "simple-rss",                     "~>1.3.1",       :require => false
 gem "snmp",                           "~>1.2.0",       :require => false
 gem "sshkey",                         "~>1.8.0",       :require => false
+gem "thin",                           "~>1.7.0",       :require => false
```

Comment:
@NickLaMuro we don't have a Gemfile.local that are for appliances.  The idea is we want appliances to have all the dependencies they need so they can switch web servers with just a configuration change to `rails_server` in our advanced settings page.

Maybe a bundler group would suffice for something like that.  We'd still need to resolve the full dependency tree but we wouldn't have to install them all.  Does that buy us much?  Maybe a little.